runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -102,6 +102,8 @@
 {
 	private static final int REFRESH_THROTTLE = 60_000; // ms
 	private static final int TICK_THROTTLE = (int) Duration.ofMinutes(10).toMillis();
+	private static final int MAX_PLAYER_COUNT = 2000;
```

Comment:
If you really want this to be expressed as a percentage of 2k you could just set it to `2000 * 0.975` (I don't see any reason to not do this calculation at compile time) but I would be find just with setting this to `1950`